.ci/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+version=$(awk '/^ENV RAPIDPRO_VERSION/ { print $3 }' rapidpro-docker/Dockerfile)
+
+echo "$version" > image-params/tag
```

Comment:
We probably want the tag to include a configurable prefix or suffix that we can set from the build so that we can separate locally-built (and thus cached) images from non-local builds.